about/about.html

```diff
@@ -0,0 +1,206 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../about.css">
+    <title>Proyect Capstone</title>
+</head>
+
+<body>
+    
+     <div id="menu" class="">
+
+         <ul id="elements" class="">
+
+             <li>
+                <a href="#intro">ABOUT</a>
+             </li>
+             
+             <li>
+                <a href="https://goo.gl/maps/TWgiApNouQkwDre69">JOIN</a>
+             </li>
+             
+             <li>
+                <a href="#logo">LOGO</a>
+             </li>
+             
+             <li>
+                <a href="#plans">PLANS</a>
+             </li>
+             
+             <li>
+                <a href="#partners">Sponsors</a>
+             </li>
+         </ul>
+     </div>
+
+     <!--NAV TOP, NAV, -->
+     <div class="intro-background">
+        <!--NAV TOP Y NAV-->
+
+        <header>
+            <nav>
+                <div class="top">
+                    <ul>
+                        <li>
+                            <a href="https://facebook.com/"><img src="../images/logo-facebook.png" alt="facebook"></a>
+                        </li>
+                        <li>
+                            <a href="https://www.instagram.com/"><img src="../images/logo-instagram.png" alt="instagram"></a>
+                        </li>
+                        <li>
+                            <a href="">English</a>
+                        </li>
+                        <li>
+                            <a href="">My page</a>
+                        </li>
+                        <li>
+                            Partners
+                        </li>
+                    </ul>
+                </div>
+
+                <div class="bottom">
+                    <button id="hamburguer-button"></button>
+                     <ul>
```

Comment:
- [ ] The hamburger button does not respond on mobile. This is so for both index and about pages. Kindly adjust this while meeting the requirements below:

### Instructions:
![image](https://github.com/maximoortelli/Capstone-Proyect/assets/50990925/0cd738dc-593b-423e-9647-4a549a399a8a)

Reply:
Thank you very much for the suggestion, the changes have been made satisfactorily 👍😊